### ISSUE TITLE
docker runs fail

### ISSUE THREAD

**Issue by fatfatson** [googleapis/gapic-generator-kotlin#38]
> docker run --rm              --mount type=bind,source="/Users/wellbye/repos/j/pn-server/proto",target=/proto              --mount type=bind,source="/Users/wellbye/repos/j/pn/app/src/main/java/com/jj/pn/grpc",target=/generated          gcr.io/kotlin-gapic/kgen

Using standard configuration...

Generating client code...
To honour the JVM settings for this build a new JVM will be forked. Please consider using the daemon: https://docs.gradle.org/4.9/userguide/gradle_daemon.html.
Daemon will be stopped at the end of the build stopping after processing
Download https://repo.maven.apache.org/maven2/io/grpc/grpc-core/maven-metadata.xml
Download https://jcenter.bintray.com/io/grpc/grpc-core/maven-metadata.xml
> Task :extractIncludeProto
> Task :extractProto
> Task :generateProto

e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (92, 25): Unresolved reference: verifycode
> Task :compileKotlin FAILED
e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (94, 21): Unresolved reference: verifycode
e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (155, 25): Unresolved reference: isadmin
e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (157, 21): Unresolved reference: isadmin
e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (170, 25): Unresolved reference: pagesize
e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (172, 21): Unresolved reference: pagesize
e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (197, 25): Unresolved reference: notifyposturl
e: /usr/src/generator/runner/build/generated/source/proto/main/client/pn/KotlinBuilders.kt: (199, 21): Unresolved reference: notifyposturl

FAILURE: Build failed with an exception.

`verifycode/isadmin/pagesize/notifyposturl` are all my message field. the proto file compiles ok for nodejs & c#.

**Comment by jbolinger** [comment]
Thanks for reporting this @fatfatson! Any chance you could share a portion of your proto so that I can take a closer look?

**Comment by fatfatson** [comment]
```
syntax = "proto3";

package pn;

message UserInfo {
  int32 id = 1;
  string phone = 2;
  string name = 3;
  string password = 5;
  string notifyPostUrl = 6;

  int32 version = 15;
}

message UserRegisterRequest {
  string id = 1;
  string phone = 2;
  string name = 3;
  string email = 4;
  string password = 5;
  string verifyCode = 6;
}

message UserRegisterResult {
  bool ok = 1;
  string msg = 2;
}

message UserLoginRequest {
  string token = 1;
  string name = 2;
  string password = 3;
}

message UserLoginResult {
  string msg = 1;
  string token = 2;
  string name = 3;
  bool isAdmin = 4;
  UserInfo info = 5;
}

message UserSetInfoRequest { string notifyPostUrl = 1; }

message AdminGetUserListRequest {
  int32 page = 1;
  int32 pageSize = 2;
}
message AdminGetUserInfo {
  int32 id = 1;
  string phone = 2;
  string name = 3;
  string notifyPostUrl = 4;
}
message AdminGetUserListResult {
  repeated AdminGetUserInfo users = 1;
  int32 total = 2;
}

message Empty {}
message PingMsg { int32 no = 1; }

service UserService {
  rpc Ping(PingMsg) returns (PingMsg);
  rpc GetUserList(AdminGetUserListRequest) returns (AdminGetUserListResult);
  rpc Register(UserRegisterRequest) returns (UserRegisterResult) {}
  rpc Login(UserLoginRequest) returns (UserLoginResult) {}
  rpc SetInfo(UserSetInfoRequest) returns (Empty) {}

  rpc TestStream(StreamRequest) returns (stream StreamResponse) {};
}

message StreamPoint {
  string name = 1;
  int64 value = 2;
  int32 hv = 3;
}

message StreamRequest { StreamPoint pt = 1; }

message StreamResponse { StreamPoint pt = 1; }
```

**Comment by jbolinger** [comment]
Thanks for sharing this @fatfatson. This was very helpful. 

The problem seems to be due to the use of `camelCase` names in the protos rather than `snake_case`. We have a pretty strict style guide internally so this hasn't been an issue, but you should be able to use whatever names you like. I'll do a fix shortly so you can use these variable names.

**Comment by jbolinger** [comment]
I believe this is now fixed (in version 0.6.0). Let me know if you are still having any trouble.

**Comment by jbolinger** [comment]
This seems to be resolved so I'm going to close this issue. Feel free to reach out if you have any more trouble.
